cmd/unused-exporter/exporter.go

```diff
@@ -272,3 +233,47 @@ func (e *exporter) Collect(ch chan<- prometheus.Metric) {
 		}
 	}
 }
+
+func getDiskLabels(d unused.Disk, v bool) []any {
+	diskLabels := []any{
```

Comment:
In all cases we're collecting a slice of `slog.Attr`, let's change the signature:
```suggestion
func getDiskLabels(d unused.Disk, v bool) []slog.Attr {
	diskLabels := []slog.Attr{
```

This should make the call in line 159 a little bit more efficient.

Reply:
I could have swear that `[]slog.Attr` would work. `[]any` is ok, `any` is actually an alias for `interface{}`.